build/bundle.go

```diff
@@ -16,6 +16,13 @@ type BundleSpec struct {
 }
 
 type Bundle struct {
+	// Version is the actors version in this bundle
 	Version actors.Version
+	// Release is the release id
 	Release string
+	// Path is the (optional) bundle path; uses the appropriate release bundle if unset
+	Path string
+	// Devlopment indicates whether this is a development version; when set, in conjunction with path,
+	// it will always load the bundle to the blockstore
```

Comment:
Can we add that it will NOT persist it in the datastore (which i think is the important part here)

Reply:
sure!

Reply:
done.